node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala

```diff
@@ -452,7 +452,7 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
 
     val phloPrice = opt[Long](
       descr = "The price of phlo for this transaction in units dust/phlo. Must be positive integer.",
-      validate = _ > 0,
+      validate = _ >= 0,
```

Comment:
This validation is only for CLI. Zero is validated on executing node so gRPC and CLI will return the same error message.

Reply:
@tgrospic it diverges from `descr`, which says `Must be positive integer`. Could you add the same comment in the code to omit confusion in future?